charts/cronjob/Chart.yaml

```diff
@@ -1,14 +1,14 @@
 apiVersion: v2
-appVersion: "v1.0.0"
+appVersion: "v1.0.1"
 home: https://github.com/ameijer/k8s-as-helm
 description: Helm Chart representing a single CronJob Kubernetes API object
 name: cronjob
-version: 1.0.0
+version: 1.0.1
 icon: https://ameijer.github.io/k8s-as-helm/icon.png
 dependencies:
 - name: lib-k8s-as-helm
-  version: 1.1.0
-  repository: file://../lib-k8s-as-helm
+  version: 1.3.0
+  repository: https://github.com/ameijer/k8s-as-helm/releases/download/lib-k8s-as-helm-1.3.0/lib-k8s-as-helm-1.3.0.tgz
```

Comment:
thanks! can you re-run helm dependency update to re-generate Chart.lock?

Reply:
`helm dependency update` has been failing because it is expecting an index.yaml file in the root of the remote repo. Is there another URL for the chart repo?

Reply:
try https://ameijer.github.io/k8s-as-helm as the repository

Reply:
thanks, that worked. I tried that previously but appended `/charts` to the repo URL.

Reply:
can you rerun failed CI checks?